improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -1403,6 +1410,49 @@ def test_manual_title(self):
         result = plugin.create_symbol_cube([self.cube])
         self.assertEqual(result.attributes["title"], target_title)
 
+    def test_blend_time_multiple_inputs(self):
+        """Test cube is constructed with appropriate blend_time with multiple
+        source cubes. Newest blend_time should be used."""
+        self.plugin.template_cube = self.cube
+        cube1 = self.cube.copy()
+        new_coord = cube1.coord("blend_time")
+        new_coord = new_coord.copy(new_coord.points + 3600)
+        cube1.replace_coord(new_coord)
+
+        result = self.plugin.create_symbol_cube([self.cube, cube1])
+
+        self.assertEqual(
+            result.coord("blend_time").cell(0).point, dt(2017, 10, 9, 2, 0),
+        )
+        self.assertEqual(len(result.coord("blend_time").points), 1)
```

Comment:
Can you add a check here that `frt + fp == vt`. This captures the current issues in the function you've written.
For completeness you should probably make changes such that  `frt + fp == vt` is true of cube1 as well.

Reply:
Test added